bool status
bool phase
int32 duty_cycle_percent